libmdnsd/1035.c

```diff
@@ -249,7 +256,8 @@ static int _rrparse(struct message *m, struct resource *rr, int count, unsigned
 	int i;
 
 	for (i = 0; i < count; i++) {
-		_label(m, bufp, &(rr[i].name));
+		if (_label(m, bufp, &(rr[i].name))
```

Comment:
Seem to be missing an ending `)` here.   Travis-CI fails to build ...

Reply:
How embarrassing. Fixed.